app/util/api.js

```diff
@@ -2,7 +2,7 @@ export function fetchNewsIDsByType(type = 'top') {
     return fetch(`https://hacker-news.firebaseio.com/v0/${type}stories.json?print=pretty`)
         .then((res) => res.json())
         .then((ids) => {
-            return ids.slice(0, 50);
+            return ids;
```

Comment:
same as above,,, no need of return keyword

Reply:
done